autodeploy/routers/predict.py

```diff
@@ -44,7 +44,7 @@ def setupRabbitMq(self, ):
 
     # connect to RabbitMQ Server.
     connection = pika.BlockingConnection(
-        pika.ConnectionParameters('localhost'))
+        pika.ConnectionParameters('127.0.0.1', port=5672))
```

Comment:
i forgot to make it configurable

Reply:
yeah .. could be in the config.yaml?

Reply:
yeah thats what i was thinking, also Ill make a base class and there this will be default